doc/Protocols/pr-meta_x-road_service_metadata_protocol.md

```diff
@@ -287,9 +289,9 @@ when retrieved through the meta-service.
 
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
-<wsdl:definitions targetNamespace="http://metadata.x-road.eu/"
+<wsdl:definitions targetNamespace="http://x-road.eu/xsd/xroad.xsd"
     xmlns:wsdl="http://schemas.xmlsoap.org/wsdl/"
-    xmlns:meta="http://metadata.x-road.eu/"
+    xmlns:meta="http://x-road.eu/xsd/xroad.xsd"
```

Comment:
The namespace prefix "meta" is now redundant since "xrd" has the same value. It does distinguish the elements defined in this wsdl, but can be confusing (the elements are still in the same namespace).

Reply:
Removed redundant namespace definition.